doc/source/trajectories/slicing_trajectories.rst

```diff
@@ -0,0 +1,65 @@
+.. -*- coding: utf-8 -*-
+.. _slicing-trajectories:
+
+====================
+Slicing trajectories
+====================
+
+MDAnalysis trajectories can be indexed to return a :class:`~MDAnalysis.coordinates.base.Timestep`, or sliced to give a :class:`~MDAnalysis.coordinates.base.FrameIterator`. 
+
+.. ipython:: python
+
+    import MDAnalysis as mda
+    from MDAnalysis.tests.datafiles import PSF, DCD
+
+    u = mda.Universe(PSF, DCD)
+    u.trajectory[4]
+
+
+Indexing a trajectory shifts the :class:`~MDAnalysis.core.universe.Universe` to point towards that particular frame, updating dynamic data such as ``Universe.atoms.positions``. 
+
+.. ipython:: python
+
+    u.trajectory.frame
```

Comment:
You lean quite a lot on the fact that `.frame` is the internal MDAnalysis index of frames (rather than the one assigned by the MD engine) for demonstrating these iterations.  Maybe  it's worth a quick `..note::` to emphasise this fact.